src/zzz_od/operation/hdd/enter_hdd_mission.py

```
@@ -72,7 +73,10 @@ def choose_mission(self) -> OperationRoundResult:
         area = self.ctx.screen_loader.get_area('HDD', '副本区域')
         result = self.round_by_ocr_and_click(self.last_screenshot, self.mission_name, area=area)
         if result.is_success:
-            return self.round_success(wait=1)
+            # 少数情况下首次点击未成功改变选中项，追加一次原地轻点以提高选择稳定性
```

Ruff (0.14.5)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)